index.js

```diff
@@ -2,7 +2,19 @@ const assert = require('assert')
 
 module.exports = cacheElement
 
-// Memoize a bel element
-// null -> null
-function cacheElement () {
+function cacheElement (fn) {
+  const store = {}
+  
+  return function render () {
+    const args = Array.from(arguments)
+    const argsAreTheSame = JSON.stringify(store.prev) === JSON.stringify(args)
```

Comment:
I suspect this might be slow for larger data sets; I think we could get away with shallow equality and triple equals. Perhaps we should allow an optional comparison function in the `cacheElement` call: `cacheElement(createElement, compare)` so that if a deep comparison or the like needs to be done it's flexible enough to be supported